changelog/README.md

```diff
@@ -0,0 +1,67 @@
+Pending changelog
+=================
+
+This will get copied to dlang.org and cleared when master gets
+merged into stable prior to 2.072
```

Comment:
The version number is going to be forgotten here. Maybe just go with "next release".